internal/pgadmin/users.go

```diff
@@ -105,7 +137,57 @@ with create_app().app_context():
 
         db.session.add(user)
         db.session.commit()
-`
+
+        # Set the cluster and host name variable.
+        clustername = "%s"
+        hostname = "%s"
+
+        # Get or create the group as necessary
+        group = (
+            ServerGroup.query.filter_by(user_id=user.id).first() or
```

Comment:
It is possible to add other groups in the UI. I believe our server group should always be the first returned, but I am testing to double check that is always the case. 

The group can be renamed temporarily, but if a change is made that adds an update server connection (currently limited to password updates or postgres port changes), it will be overwritten. 

In testing, the pgAdmin console does not allow the server group to be removed.